README.md

```diff
@@ -157,6 +157,106 @@ Gets the permission information of a record
 
 ---
 
+# Append data to Linnia
+
+In order to append data to Linnia your web3 instance need to be able to sign with the private key of the owner of the data.
+
+## linnia.addRecord
+
+```javascript
+linnia.addRecord(dataHash, metadata, dataUri, ethParams);
+```
+
+Add record to Linnia
+
+### Parameters
+
+1. `String` - The data hash. Hash of the plain text data + metadata
+2. `String|Object` - The metadata of the record. [Click here to read more about the metadata](https://github.com/ConsenSys/linnia-resources/blob/master/METADATA.md)
+3. `String` - The dataUri, link to the data (eg. the IPFS hash)
+4. `Object` - The ethParams ethereum account params. 
+
+### Returns
+
+`Promise<Record>` - A promise when resolved returns the record object that was stored.
+
+### Append data using Linnia Js in the browser
+
+ If you are using a Linnia Js in the browser you can create the Linnia instance with the web3 object from Metamask and append a record the following way:
+
+### Example
+
+```javascript
+const Linnia = require("@linniaprotocol/linnia-js");
+const linnia = new Linnia(web3);
+
+const dataHash = "0xcc85fc3d763b9a1d83e4386b37b4b0f3daf9881638ba8b7db0c501c417acb689";
+const metadata = {
+    dataFormat: "json",
+    domain: "social media",
+    storage: "IPFS",
+    encryptionScheme: "x25519-xsalsa20-poly1305",
+    encryptionPublicKey: "hQYhHJpzZH/tGhz1wtqSjkL17tJSnEEC4yVGyNTHNQY=",
+    linniajsVersion: "0.1.4",
+    ownerName: "John Linnia",
+    providerName: "SocialMedia",
+    providerEthereumAddress: "0x349e31e92027f86b0ffeb5cd5e07003c7f229872",
+    keywords: [ "socialmedia", "friends list", "people" ],
+}
+const dataUri = "QmSg3jCiroFERczWdpFJUau5CofHfMKCSm5vZXSzn7sZGW";
+const ethParams = {
+    from: "0xb717d7adf0d19f5f48bb7ff0030e30fcd19eed72", gas: 500000, gasPrice: 20000000000
+};
+
+const record = await linnia.addRecord(dataHash, metadata, dataUri, ethParams);
+```
+
+In the example above the dataUri is the IPFS Hash where the file was stored.
+
+### Append data using Linnia Js outside of the browser
+
+ If you are using a Linnia Js ouside of the browser. In order to add a file you need to generate a web3 instance that can handle the private key of the owner of the file.
```

Comment:
Get rid of the first sentence. It's a dupe of the title

Reply:
Also, was there a good article you came across during your research that we can link to here? Would be nice to link to something that explains what you mean by the last sentence in detail

Reply:
Maybe we can try to explain it better. The thing is someone has to handle the keys. If you use the browser it can be Metamask.

Reply:
I removed the first sentence